gmp.spec

```diff
@@ -1,2 +1,2 @@
-### RPM external gmp 4.2.1
-Source: ftp://mirrors.kernel.org/gnu/%n/%n-%realversion.tar.gz
+### RPM external gmp 6.1.2
```

Comment:
We don't use this SPEC. GMP is distributed as part of GCC toolchain package itself. The externals should use `gmp-static` package, which does not provide shared library, just static one.